interface/forms/requisition/new.php

```diff
@@ -0,0 +1,363 @@
+<?php
+/**
+ *  Lab Requisition Form
+ *
+ * @package OpenEMR
+ * @link    http://www.open-emr.org
+ * @author  Sherwin Gaddis <sherwingaddis@gmail.com>
+ * @copyright Copyright (c) 2016-2017 Sherwin Gaddis <sherwingaddis@gmail.com>
+ * @license https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ */
+
+use OpenEMR\Core\Header;
+
+include_once("../../globals.php");
```

Comment:
Make this require_once

Reply:
Also, I think I caused a misunderstanding here. You previously had also had:
``````
include_once("$srcdir/api.inc");
require_once("$srcdir/patient.inc");
require_once("$srcdir/options.inc.php");
require_once("$srcdir/htmlspecialchars.inc.php");
require_once("$srcdir/lab.inc");
``````
And I meant to say that htmlspecialchars.inc.php is not needed here since it is autoloaded. The others are not autoloaded, so if you are using them in this script, then need to add them back (and ensure use require_once rather than include_once).